packages/assets-controllers/src/NftDetectionController.test.ts

```diff
@@ -294,19 +294,21 @@ describe('NftDetectionController', () => {
     testNftDetection.startPollingByNetworkClientId('mainnet', {
       address: '0x1',
     });
+    await Promise.all([jest.advanceTimersByTime(0), Promise.resolve()]);
```

Comment:
Nit: Does it make sense to copy over `flushPromises` from other packages? It could be nice to have these tests look similar to other tests for other packages so it's easier to compare them.

Reply:
done https://github.com/MetaMask/core/pull/1874/commits/fa72da746a3bc50a6e2e49afbf94ec85b5135fe8